### PR TITLE
OADP-1378: Check OADP 1.1.2 documentation for references to earlier versions

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves: https://issues.redhat.com/browse/OADP-1378
OADP 1.1.2
OCP 4.9+
Preview:
https://56956--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-features-plugins.html#oadp-support-for-ibm-power-and-ibm-z
https://56956--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-features-plugins.html#oadp-ibm-power-test-matrix_oadp-features-plugins
https://56956--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.html#oadp-using-data-mover-for-csi-snapshots_backing-up-applications

### CODE REVIEW

File: modules/oadp-using-data-mover-for-csi-snapshots.adoc [hunk 1]
```diff
@@ -9,7 +9,7 @@
 :FeatureName: Data Mover for CSI snapshots
 include::snippets/technology-preview.adoc[]
 
-The OADP 1.1.0 Data Mover enables customers to back up container storage interface (CSI) volume snapshots to a remote object store.  When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs. The OADP 1.1.0 Data Mover solution uses the Restic option of VolSync.
+The OADP 1.1.2 Data Mover enables customers to back up container storage interface (CSI) volume snapshots to a remote object store.  When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs. The OADP 1.1.0 Data Mover solution uses the Restic option of VolSync.
```
Comment (PrasadJoshi12) [inline modules/oadp-using-data-mover-for-csi-snapshots.adoc:12]:
```suggestion
The OADP 1.1.2 Data Mover enables customers to back up container storage interface (CSI) volume snapshots to a remote object store.  When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs. The OADP 1.1.2 Data Mover solution uses the Restic option of VolSync.
```

Comment (sbeskin-redhat) [reply modules/oadp-using-data-mover-for-csi-snapshots.adoc]: @PrasadJoshi12 
Corrected

**Review by PrasadJoshi12 — APPROVED** [review]

**Comment by anarnold97** [comment]
@weshayutin @kaovilai please can I ask for a Dev's check on this? thanks

File: modules/oadp-ibm-power-test-support.adoc [hunk 2]
```diff
@@ -6,4 +6,4 @@
 [id="oadp-ibm-power-test-matrix_{context}"]
 = OADP support for target backup locations using IBM Power
 
-IBM Power running with {product-title} 4.11 and OpenShift API for Data Protection (OADP) 1.1.0 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running IBM Power with {product-title} 4.11 and OADP 1.1.0 against all non-AWS S3 backup location targets as well.
+IBM Power running with {product-title} 4.11 and 4.12 and OpenShift API for Data Protection (OADP) 1.1.2 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running IBM Power with {product-title} 4.11 and 4.12 and OADP 1.1.2 against all non-AWS S3 backup location targets as well.
```
Comment (mjpytlak) [inline modules/oadp-ibm-power-test-support.adoc:9]:
```suggestion
IBM Power running with {product-title} 4.11 and 4.12, and OpenShift API for Data Protection (OADP) 1.1.2 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running IBM Power with {product-title} 4.11 and 4.12, and OADP 1.1.2 against all non-AWS S3 backup location targets as well.
```

File: modules/oadp-ibm-z-test-support.adoc [hunk 3]
```diff
@@ -6,4 +6,4 @@
 [id="oadp-ibm-z-test-support_{context}"]
 = OADP testing and support for target backup locations using {ibmzProductName}
 
-{ibmzProductName} running with {product-title} 4.11 and OpenShift API for Data Protection (OADP) 1.1.0 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibmzProductName} with {product-title} 4.11 and OADP 1.1.0 against all non-AWS S3 backup location targets as well.
+{ibmzProductName} running with {product-title} 4.11 and 4.12 and OpenShift API for Data Protection (OADP) 1.1.2 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibmzProductName} with {product-title} 4.11 and 4.12 and OADP 1.1.2 against all non-AWS S3 backup location targets as well.
```
Comment (mjpytlak) [inline modules/oadp-ibm-z-test-support.adoc:9]:
```suggestion
{ibmzProductName} running with {product-title} 4.11 and 4.12, and OpenShift API for Data Protection (OADP) 1.1.2 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibmzProductName} with {product-title} 4.11 and 4.12, and OADP 1.1.2 against all non-AWS S3 backup location targets as well.
```

File: modules/oadp-using-data-mover-for-csi-snapshots.adoc [hunk 4]
```diff
@@ -9,7 +9,7 @@
 :FeatureName: Data Mover for CSI snapshots
 include::snippets/technology-preview.adoc[]
 
-The OADP 1.1.0 Data Mover enables customers to back up container storage interface (CSI) volume snapshots to a remote object store.  When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs. The OADP 1.1.0 Data Mover solution uses the Restic option of VolSync.
+The OADP 1.1.2 Data Mover enables customers to back up container storage interface (CSI) volume snapshots to a remote object store.  When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs. The OADP 1.1.2 Data Mover solution uses the Restic option of VolSync.
```
Comment (mjpytlak) [inline modules/oadp-using-data-mover-for-csi-snapshots.adoc:12]:
```suggestion
The OADP 1.1.2 Data Mover enables customers to back up container storage interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs. The OADP 1.1.2 Data Mover solution uses the Restic option of VolSync.
```

**Review by mjpytlak — COMMENTED** [review]
@sbeskin-redhat A few items for consideration.

File: modules/oadp-ibm-power-test-support.adoc (hunk 2 above)
Comment (sbeskin-redhat) [reply modules/oadp-ibm-power-test-support.adoc]: @mjpytlak 
Done

File: modules/oadp-ibm-z-test-support.adoc (hunk 3 above)
Comment (sbeskin-redhat) [reply modules/oadp-ibm-z-test-support.adoc]: @mjpytlak 
Done

File: modules/oadp-using-data-mover-for-csi-snapshots.adoc (hunk 4 above)
Comment (sbeskin-redhat) [reply modules/oadp-using-data-mover-for-csi-snapshots.adoc]: @mjpytlak 
Done

**Comment by mjpytlak** [comment]
@sbeskin-redhat Please squash your commits.

**Comment by sbeskin-redhat** [comment]
@mjpytlak 
Squashed. Please merge

**Comment by mjpytlak** [comment]
@sbeskin-redhat Please resolve the conflicts in 4.9 and 4.10 and let me know when the manually cherry picks are in place. I will be off the better part of next week, so you might have to hit up the merge squad again.

**Pull request merged**